contracts/oracle_controller/library.cairo

```diff
@@ -81,379 +79,396 @@ end
+    ):
+        OracleController_publisher_registry_address_storage.write(publisher_registry_address)
+        _set_keys_decimals(keys_decimals_len, keys_decimals, 0)
+        return ()
+    end
 
-func OracleController_initialize_oracle_controller{
-    syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr
-}(publisher_registry_address : felt, keys_decimals_len : felt, keys_decimals : KeyDecimalStruct*):
-    OracleController_publisher_registry_address_storage.write(publisher_registry_address)
-    _OracleController_set_keys_decimals(keys_decimals_len, keys_decimals, 0)
-    return ()
-end
+    #
+    # Getters
+    #
 
-#
-# Getters
-#
+    func get_publisher_registry_address{
+        syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr
+    }() -> (publisher_registry_address : felt):
+        let (publisher_registry_address) = OracleController_publisher_registry_address_storage.read(
+            )
+        return (publisher_registry_address)
+    end
 
-func OracleController_get_publisher_registry_address{
-    syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr
-}() -> (publisher_registry_address : felt):
-    let (publisher_registry_address) = OracleController_publisher_registry_address_storage.read()
-    return (publisher_registry_address)
-end
+    func get_active_oracle_implementation_addresses{
+        syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr
+    }() -> (oracle_addresses_len : felt, oracle_addresses : felt*):
+        alloc_locals
+
+        let (total_oracle_addresses_len) = OracleController_oracle_implementations_len_storage.read(
+            )
+        let (local oracle_addresses) = alloc()
 
-func OracleController_get_active_oracle_implementation_addresses{
-    syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr
-}() -> (oracle_addresses_len : felt, oracle_addresses : felt*):
-    alloc_locals
+        if total_oracle_addresses_len == 0:
+            return (total_oracle_addresses_len, oracle_addresses)
+        end
 
-    let (total_oracle_addresses_len) = OracleController_oracle_implementations_len_storage.read()
-    let (local oracle_addresses) = alloc()
+        let (oracle_addresses_len, oracle_addresses) = build_active_oracle_implementation_addresses(
+            total_oracle_addresses_len, oracle_addresses, 0, 0
+        )
 
-    if total_oracle_addresses_len == 0:
-        return (total_oracle_addresses_len, oracle_addresses)
+        return (oracle_addresses_len, oracle_addresses)
     end
 
-    let (
-        oracle_addresses_len, oracle_addresses
-    ) = OracleController_build_active_oracle_implementation_addresses(
-        total_oracle_addresses_len, oracle_addresses, 0, 0
-    )
+    func get_oracle_implementation_status{
+        syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr
+    }(oracle_implementation_address : felt) -> (
+        oracle_implementation_status : OracleController_OracleImplementationStatus
```

Comment:
Can we move struct definition into the namespace as well?

Reply:
Since we don't need them in the storage vars, I'm imagining we could potentially remove all `OracleController_` prefixes since there's no concern about collisions.  I would like to find a way to be more confident that this prefix is only necessary if there's conflicts in storage_var names within our codebase, and not necessary otherwise.

I think this is a good topic for the audit.

Reply:
https://github.com/OpenZeppelin/cairo-contracts/blob/main/src/openzeppelin/token/erc20/library.cairo#L32

although OpenZeppelin is still using this convention so maybe we should use their implementation as the standard

Reply:
Sgtm, let's stick with OZ standard for now